zulipterminal/ui_tools/boxes.py

```diff
@@ -1099,14 +1099,12 @@ def soup2markup(cls, soup: Any, metadata: Dict[str, Any],
                 # TODO: Address 12-hour format support with application-wide
                 # support for different formats.
                 time_string = local_time.strftime('%a, %b %-d %Y, %-H:%M (%Z)')
+                source_text = f"Original text was {raw_text.strip()}"
+                metadata['time_mentions'].append((time_string, source_text))
```

Comment:
I'd suggest squashing this with the "shorten lines" commit that I mentioned floating to just after the first commit in the other comment - the rename is in the first commit, and this is a simplification enabled by that first commit.

You have moved code, which is a minor extra change we could include too.

Reply:
:+1: